Tests/MapboxMapsTests/Foundation/MapViewTests.swift

```diff
@@ -261,91 +261,6 @@ final class MapViewTests: XCTestCase {
         XCTAssertEqual(runner.updateStub.invocations.count, 1)
     }
 
-    func testAppLifecycleNotificationSubscribedWhenDidMoveToNewWindow() {
```

Comment:
move this to another test class so I can manage Mock Bundle with mock scene support better.